shared/actions/login/index.js

```diff
@@ -419,13 +434,33 @@ export function addANewDevice () : AsyncAction {
         isMobile ? Constants.codePageDeviceRoleExistingPhone : Constants.codePageDeviceRoleExistingComputer
     })
 
-    const incomingMap = makeKex2IncomingMap(dispatch, getState)
+    const onBack = response => {
```

Comment:
if things cancel or work we basically want to refresh devices and go back to the root